.goreleaser.yaml

```diff
@@ -0,0 +1,48 @@
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - id: capo
+    binary: capo
+    dir: ./cmd/capo
+    ldflags:
+      - -extldflags "-static" -s -w -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser -X main.Version={{.Version}} -X main.Revision={{.ShortCommit}}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+    goarm:
+      - "7"
```

Comment:
I honestly have no idea if this is correct or how to check or what are the implications. Why is specifically ARMv7 here?